Jenkinsfiles/utils.groovy

```diff
@@ -12,7 +12,10 @@ def get_commit_tag() {
     return env.GIT_COMMIT.take(7)
 }
 
-def get_target_name() {
+def get_target_name(target_name='') {
+    if (target_name != null && !target_name.trim().isEmpty()){
+        return target_name
+    {
```

Comment:
I just noticed that this is missing the closing bracket `}` (you have another `{` instead), but it can be simplified as well, since a `null` value as well as an empty string are both **falsy** in Groovy:
```suggestion
    if (target_name) {
        return target_name
    }
```

Reply:
Good catch! Fix in 4befc0ff6d6d953b8149da901e9902770922c1c1